src/components/ha-menu-button.ts

```diff
@@ -130,6 +130,7 @@ class HaMenuButton extends LitElement {
     return css`
       :host {
         position: relative;
+        margin-right: 12px;
```

Comment:
This should not be in `ha-menu-button` the user of this element should set the margin.